USER_MANUAL.md

```diff
@@ -628,7 +673,543 @@ The measured total site level emission rate on January 30th is assumed to have l
+
+**Notes on acquisition:** No data acquisition required.
+
+**Notes of caution:** The instant threshold should be set above the [follow-up threshold](#threshold), as this is the minimum emission rate that a site must reach to be flagged.
+
+Follow-up thresholds are explored in detail in Fox et al. 2021\. Choosing follow-up rules is complex and work practices should be developed following extensive analysis of different scenarios. It is important to understand how follow-up thresholds and follow-up ratios interact, especially if both are to be used in the same program. Note that follow-up thresholds are similar to minimum detection limits but that the former is checked against the measured emission rate (which is a function of quantification error) while the latter is checked against the true emission rate.
+
+#### &lt;interaction_priority&gt;
+
+**Data type:** String
+
+**Default input:** "threshold"
+
+**Description:** Specifies which algorithm to run first on candidate sites when determining which to flag. The following are the valid options for this parameter:
+
+- `threshold`: The proportion of sites to follow up with will be taken from all sites over the threshold
+- `proportion`: The proportion of sites will be taken from the candidate sites, then from those sites follow-up will occur at sites above the threshold.
+
+**Notes on acquisition:** No data acquisition required.
+
+**Notes of caution:** N/A
+
+#### &lt;proportion&gt;
+
+**Data type:** Numeric
+
+**Default input:** 1.0
+
+**Description:** A ratio (0 to 1.0) defines the proportion or percentage of flagged sites that will receive follow-ups. For example, if the follow-up ratio is 0.5, the top 50% of flagged sites (ranked by measured emission rate) will receive follow-up. Candidate flagged sites have already been checked against the minimum detection limit.
+
+**Notes on acquisition:** No data acquisition required.
+
+**Notes of caution:** Specifies which measured emission rates to use when identifying flagged candidate sites for follow-up. This is relevant for individual sites that have had multiple independent measurements during the given period, resulting in the sites being added to a candidate pool of potentially flagged sites. The following are the three options available for this parameter:
+
+#### &lt;redundancy_filter&gt;
+
+**Data type:** String
+
+**Default input:** "recent"
+
+**Description:** Specifies which measured emission rates to utilize when identifying flagged candidate sites for follow-up. This is relevant for individual sites that have had multiple independent measurements in the given period where the candidate flagged site pool has been accumulating. The following are the 3 options available for this parameter:
+
+- `recent`(default): The most recent measurement for a given site in the "flagged pool" will be used to check the follow-up threshold and proportion.
```

Comment:
Can we double check that recent is still the default behavior with V4

Reply:
This is still the behavior.